Add onboarding guide and checklist (#6)
voiio/voiio.github.io: onboarding.md

### Onboarding
Congratulations and welcome to voiio!

We are happy to have you in the team, we know that starting a new job can be both exciting and a little bit daunting. This guide is meant to help you get settled.

### Checklist

> To make sure you get a smooth onboarding and enjoy it, your buddy will be there to help you accomplish the following checklist.

Reviewer: You are referencing the buddy here, without prior mentioning it. Maybe you want to add a section about the buddy first?

**We believe that the best way is to get started gradually, and work on issues of increasing depth, This is the progression of your first tasks:**

1. A documentation ticket to improve our documentation and to get you acquainted with our development and deployment pipeline.

2. Fix a small technical issue using some technology you already know. This will be close to vanilla Python/Django or Javascript/Vue.js.

3. A more challenging technical issue, this time you will get to work with a part of our stack that is new to you.

4. Now you are ready to dive into our business domain and solve a substantive issue for one of our products and its users.

#### First day
* Setup computer and install the development environment.
* Get access to all necessary [accounts](accounts.md).

> * Push your first commit to production.

Reviewer: ```suggestion
* Deploy your first commit on production.
```

Reviewer: The last task of the first day should be to extend or update the onboarding guide.

Reviewer: Never mind, found it. However, I don't want people to forget about it. IMHO it should be an explicit first-day task.

#### First week
* 15 min daily checkins with buddy.

> * Intro to Frontend with Sebastian.

Reviewer: Maybe use GitHub handles here to link to the actual GH profile.

* Intro to Metabase with Rust.
* Intro to Google Analytics with Mos.

#### First Month

> * We do a team event all the tech team together.

Reviewer: ```suggestion
* We, the entire tech team, do your first team event together.
```
